src/runtime/runtime.js

```diff
@@ -881,6 +885,16 @@ export class ConfiguredRuntime {
   eventManager() {
     return this.eventManager_;
   }
+
+  /**
+   * Note that this returns Logger instead of LoggerApi for internal logging
```

Comment:
nit: this comment is confusing to callers. What you are trying to say that it returns the implementation of LoggerApi?

/**
 * Returns the Logger module that implements the LoggerApi
 */

Reply:
It was telling you that it was returning Logger, which had a slightly different API than LoggerAPI because we had the internal use function on there to logSwgEvents.  We took that out though making the comment out of date and the override unnecessary.  Removed.